tests/manage/mcg/test_bucket_deletion.py

```diff
@@ -26,55 +27,79 @@ class TestBucketDeletion:
     Test bucket Creation Deletion of buckets
     """
     @pytest.mark.parametrize(
-        argnames="amount,interface",
+        argnames="amount,interface,bucketclass_dict",
         argvalues=[
             pytest.param(
-                *[3, 'S3'],
+                *[3, 'S3', None],
                 marks=[pytest.mark.polarion_id("OCS-1939"), tier1, acceptance]
             ),
             pytest.param(
-                *[3, 'CLI'],
+                *[3, 'CLI', None],
                 marks=[tier1, acceptance, pytest.mark.polarion_id("OCS-1940")]
             ),
             pytest.param(
-                *[3, 'OC'],
+                *[3, 'OC', None],
                 marks=[tier1, acceptance, pytest.mark.polarion_id("OCS-1299")]
             ),
             pytest.param(
-                *[100, 'S3'],
+                *[100, 'S3', None],
                 marks=[
                     pytest.mark.skip(ERRATIC_TIMEOUTS_SKIP_REASON),
                     performance, pytest.mark.polarion_id("OCS-1865")
                 ]
             ),
             pytest.param(
-                *[100, 'OC'],
+                *[100, 'OC', None],
                 marks=[
                     pytest.mark.skip(ERRATIC_TIMEOUTS_SKIP_REASON),
                     performance, pytest.mark.polarion_id("OCS-1915")
                 ]
             ),
             pytest.param(
-                *[1000, 'S3'],
+                *[1000, 'S3', None],
                 marks=[
                     pytest.mark.skip(ERRATIC_TIMEOUTS_SKIP_REASON),
                     performance, pytest.mark.polarion_id("OCS-1866")
                 ]
             ),
             pytest.param(
-                *[1000, 'OC'],
+                *[1000, 'OC', None],
                 marks=[
                     pytest.mark.skip(ERRATIC_TIMEOUTS_SKIP_REASON),
                     performance, pytest.mark.polarion_id("OCS-1916")
                 ]
             ),
+            pytest.param(
+                *[1, 'OC', {
+                    'interface': 'OC',
+                    'backingstores': {
+                        'pv': [(1, 50, DEFAULT_STORAGECLASS_RBD)]
+                    }
+                }],
+                marks=[tier1, pytest.mark.polarion_id("OCS-2354")]
+            ),
+            pytest.param(
+                *[1, 'CLI', {
+                    'interface': 'CLI',
+                    'backingstores': {
+                        'pv': [(1, 50, DEFAULT_STORAGECLASS_RBD)]
+                    }
+                }],
+                marks=[tier1, pytest.mark.polarion_id("OCS-2354")]
+            )
         ]
     )
-    def test_bucket_delete(self, verify_rgw_restart_count, mcg_obj, bucket_factory, amount, interface):
+    def test_bucket_delete(self, verify_rgw_restart_count, mcg_obj, bucket_class_factory,
```

Comment:
Break down the parameters into separate lines from the parentheses in order to follow our [coding guidelines](https://ocs-ci.readthedocs.io/en/latest/coding_guidelines.html)